docs/API.md

```diff
@@ -47,73 +54,135 @@ function myApplicationLogic() {
 }
 ```
 
-##### Scope Manager
+##### Scope
```

Comment:
Did you purposely change it from Scope Manager to Scope? It seems to still be describing the scope manager

Reply:
The initial implementation had a scope manager that would create multiple scopes (still used in noop cases) per OT spec.  This was restructured as a global scope utility for improved cps support: https://github.com/signalfx/signalfx-nodejs-tracing/blob/master/src/scope/new/scope.js#L9.  Since it's an implementation of a modified OpenTracing scope manager but solely within the `Scope` object, I've attempted to unify the notions.